node_modules/oae-core/members/js/members.js

```diff
@@ -46,7 +46,7 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
 
             var url = '/api/group/' + widgetData.context.id + '/members';
             if (query) {
-                url = '/api/search/members/'  + widgetData.context.id;
+                url = '/api/search/group-members-library/'  + widgetData.context.id;
```

Comment:
From a product development standpoint we have a general convention that breaking changes require at least a minor version bump. So those with their own OAE deployments should be wary of API changes both internal and external and test accordingly prior to releasing a minor version upgrade.

From a hosting provider stand-point, we do not have a policy on releasing non-backwards-compatible API changes to production. It seems we should at least get acknowledgement from GeorgiaTech before releasing such a change, which is simple enough. I'm not aware of any other production API consumers at this point, but we can probably get an idea of external usage from our web analytics, actually.

@nicolaasmatthijs do you have any idea off hand how much usage comes from what looks like direct API access, if any?